docs/configuration-reference.md

```diff
@@ -2046,6 +2046,27 @@ receivers:
 # If a match is found then relabeling rules are applied.
 scrape_configs:
   - [<scrape_config>]
+  
+# spanmetrics supports aggregating Request, Error and Duration (R.E.D) metrics from span data.
+# https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.21.0/processor/spanmetricsprocessor/README.md.
+# spanmetrics generates two metrics from spans and exposes them to an HTTP endpoint (prometheus exporter).
```

Comment:
it's not clear to me if this is a remote write configuration or a scrape endpoint configuration. i'm guessing this just opens an http server on a port and exposes the metrics at `/metrics` ?

either way can we get a little more verbiage to clarify?

Reply:
Exactly. Can you take a another look?